File: src/main/java/com/wanderers/hotelier_webservices/rest/exception/UnauthorizedHotelierException.java
package com.wanderers.hotelier_webservices.rest.exception;

public class UnauthorizedHotelierException extends RuntimeException {

    private static final long serialVersionUID = 1L;

    /**
     * Class constructor
     *
     * @param message String value of Exception
     */
    public UnauthorizedHotelierException(String message) {
        super(message);
    }
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/rest/exception/UnauthorizedHotelierException.kt
package com.wanderers.hotelier_webservices.rest.exception

class UnauthorizedHotelierException
/**
 * Class constructor
 *
 * @param message String value of Exception
 */
    (message: String?) : RuntimeException(message) {
    companion object {
        private const val serialVersionUID = 1L
    }
}